var/spack/repos/builtin/packages/procps/package.py

```diff
@@ -27,22 +35,22 @@ class Procps(AutotoolsPackage):
     depends_on("pkgconfig@0.9.0:", type="build")
     depends_on("dejagnu", type="test")
     depends_on("iconv")
+    depends_on("gettext", type="build")
     depends_on("gettext", when="+nls")
     depends_on("ncurses")
 
     conflicts("platform=darwin", msg="procps is linux-only")
 
+    # Need to tell the build to use the tools it already has to find
+    # libintl (if appropriate).
+    patch("libintl-3.3.14.patch", when="@3.3.14:3.3")
+    patch("libintl-4.0.0.patch", when="@=4.0.0")
```

Comment:
```suggestion
    patch("libintl-4.0.0.patch", when="@4.0.0")
```

From a quick grep this seems to be the preferred syntax when a specific match is required?

Reply:
Your suggested patch would match (e.g.) 4.0.0.3; as specified it matches 4.0.0 and nothing else. The `@=` notation is relatively new.

Reply:
Oh, I didn't know about that syntax. Thanks!